src/entities/player.lua

```diff
@@ -1,8 +1,13 @@
 local assets = require("assets")
+local base = require('entities.base')
+
 local player = {}
+setmetatable(player, {__index=base})
```

Comment:
Is this the what is effectively doing the "inheritance"?

Reply:
lol yeah.  think of javascript prototypes.

Reply:
Hahaha indeed!